app/src/debug/kotlin/io/homeassistant/companion/android/developer/catalog/HAComposeCatalogScreen.kt

```diff
@@ -255,6 +260,102 @@ private fun LazyListScope.textStyles() {
     }
 }
 
+private fun LazyListScope.input() {
+    catalogSection(title = "Input") {
+        Column(
+            modifier = Modifier.fillMaxWidth(),
+        ) {
+            var value1 by remember { mutableStateOf("") }
+            var value2 by remember { mutableStateOf("") }
+            var value3 by remember { mutableStateOf("") }
+            var value4 by remember { mutableStateOf("") }
+            var value5 by remember { mutableStateOf("") }
+
+            HATextField(
+                value = value1,
+                onValueChange = { value1 = it },
+                trailingIcon = {
+                    if (value1.isNotBlank()) {
+                        IconButton(onClick = { value1 = "" }) {
+                            Icon(
+                                imageVector = Icons.Default.Close,
+                                contentDescription = null,
+                            )
+                        }
+                    }
+                },
+            )
+            HATextField(
+                value = value2,
+                onValueChange = { value2 = it },
+                placeholder = {
+                    Text(
+                        text = "Placeholder",
+                        style = HATextStyle.UserInput.copy(color = Color.Unspecified),
+                    )
+                },
+            )
+            HATextField(
+                value = value3,
+                onValueChange = { value3 = it },
+                label = {
+                    Text(
+                        text = "Label",
+                        style = HATextStyle.UserInput.copy(color = Color.Unspecified),
+                    )
+                },
+            )
+            HATextField(
+                value = value4,
+                onValueChange = { value4 = it },
+                label = {
+                    Text(
+                        text = "Label",
+                        style = HATextStyle.UserInput.copy(color = Color.Unspecified),
+                    )
+                },
+                placeholder = {
+                    Text(
+                        text = "Placeholder",
+                        style = HATextStyle.UserInput.copy(color = Color.Unspecified),
+                    )
+                },
+            )
+            HATextField(
+                value = value5,
+                onValueChange = { value5 = it },
+                leadingIcon = {
+                    Icon(
+                        imageVector = Icons.Outlined.Rectangle,
```

Comment:
Maybe use info like the screenshot text? I tapped it more than once while using the catalog thinking it was a checkbox 😅